css/index.css

```diff
@@ -95,6 +124,75 @@ p {
     margin-bottom: 0;
 }
 
+/* About Page */
+
+.about-intro {
+    padding-bottom: 20px;
+    margin-bottom: 20px;
+    border-bottom: 1px dashed black;
+}
+
+.about-container {
+    padding: 0 1%;
+}
+
+.about-main {
+    display: flex;
+    flex-flow: row wrap;
+    justify-content: space-between;
+}
+
+.about-main .about-box-a {
+    width: 48%;
+    padding: 0 1%;
+    margin-bottom: 30px;
+    border-bottom: 1px dashed black;
+}
+
+.about-main .about-box-b {
+    width: 48%;
+    padding: 0 1%;
+    flex-flow: row wrap;
+    margin-bottom: 30px;
+    border-bottom: 1px dashed black;
+}
+
+.about-main .about-box-c {
+    width: 48%;
+    padding: 0 1%;
+    flex-flow: row wrap;
+    margin-bottom: 30px;
+    border-bottom: 1px dashed black;
+}
+
+.about-main .about-box-d {
+    width: 48%;
+    padding: 0 1%;
+    flex-flow: row wrap;
+    margin-bottom: 30px;
+    border-bottom: 1px dashed black;
+}
+
+.about-button {
```

Comment:
Since you only have one style for your buttons, there's no need to create an `about-button` class. You can simply target the `button` tag like so: 
```
button {
  /* styles here */
}
```